src/lib.rs

```diff
@@ -1229,6 +1252,7 @@ impl<T: Counter> Histogram<T> {
     ///
     /// Note that the return value is capped at `u64::max_value()`.
     pub fn median_equivalent(&self, value: u64) -> u64 {
+        // TODO isn't this just saturating?
```

Comment:
Actually, it looks like this should never overflow, so I'll use checked_add (and a different PR, just in case that discussion might take us in a different direction from this PR)